Tests/Unit/MiniAppViewTests.swift

```diff
@@ -0,0 +1,29 @@
+import Quick
+import Nimble
+@testable import MiniApp
+
+class MiniAppViewTests: QuickSpec {
+
+    override func spec() {
+        describe("Mini App view") {
+            context("when initialized with valid parameters") {
```

Comment:
What about invalid parameters?

Reply:
I need to use failable initializers then. Working on it

Reply:
Done